promise-it-wont-hurt-solutions-master/src/07-promise-after-promise.js

```diff
@@ -0,0 +1,7 @@
+var promise = new Promise(function (fulfill, reject) {
+    return query.first(second);
```

Comment:
This exercise was confusing I feel. first() and second() are already methods that are available to us, which is unclear in the explanation. The intention this exercise is to execute first(), then second(), then console.log.

This code works.
first().then(second).then(console.log);